### ISSUE TITLE
A11y_ML .NET Model Builder_ Image Classification-Data _ Usability: In narrator scan mode when user navigates to "example folder structure" using up/down arrow keys it would be better if narrator reads the levels along with reading the content

### ISSUE THREAD

**Issue by SaisreeDandamudi** [dotnet/machinelearning-modelbuilder#1154]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#A11yUsable; #A11yTCS; #Win32; #DesktopApp; #Feature-ML.NETModelBuilder-Win32-Nov2020; #A11ySev4; #Benchmark; #Visual Studio; #Screen Reader; #Narartor; #NVDA;

### Environment Details:
Microsoft Visual Studio 2019 Preview Version 16.8.0 Preview 6.0

### Screen Readers:
Narrator
NVDA

### Repro Steps:

1.Start Screen Reader
2.Open Visual Studio Enterprise 2019 Preview 
3. Create a new .Net Core Console App (C#) 
4. Tab to "Solution Explorer" and right click on the project (Shift + F10) and then from the expanded list Select "Add" and then from the list items select "Machine learning"
5. Tab till "Image Classification" scenario and select the scenario
6. Select any of the environment and tab till "next step" button and press enter
7. Enable scan mode and navigate till example folder structure" and verify how narrator is reading the content

### Actual Result:
In narrator scan mode when user navigates to "example folder structure" using up/down arrow keys narrator is just reading as "Label1", "Image1" & "Image2" etc. but not reading the levels 

### Expected Result:
In narrator scan mode when user navigates to "example folder structure" using up/down arrow keys it would be better if narrator reads the levels along with reading the content

### User Impact:
It will more useful and screen reader users will understand more clearly the structure if narrator reads the levels

### Attachment for Reference

[1154_ML .NET__Narrator not reading the levels for format.pptx](https://github.com/dotnet/machinelearning-modelbuilder/files/5670883/1154_ML.NET__Narrator.not.reading.the.levels.for.format.pptx)

**Comment by beccamc** [comment]
@briacht and @romalpani Any thoughts here on what the language should be? The problem is that when going through the Images example it's hard to tell what's happening via screen reader. Saisree suggested Level 1, Level 2, etc. "Images, Level 1" "Label1, Level 2". Any thoughts? 

![image](https://user-images.githubusercontent.com/9122518/103318494-2d8d1280-49f4-11eb-8e56-56ef27c408e7.png)

**Comment by romalpani** [comment]
@beccamc 
This is an interesting one. Could we do what the narrator normally does for a folder structure?
I used the 'Narrator app' in windows to narrate a similar folder structure. I made some tweaks to what I heard, but what do you think about something like this?

"Tree View ... Images ... expanded
  Label 1 ... One of three ... expanded
       Image 1 ... One of two
       Image 2 ... Two of two
  Label 2 ... Two of three
  Label 3 ... Three of three"

**Comment by beccamc** [comment]
@romalpani That sounds good! How does the user know they're back up a level on "Label 2"? Is it because Image 2 is said as 2/2?

**Comment by romalpani** [comment]
Yes, I think that the count helps in giving the user an idea of what level they are at, in addition to when it reads out the "Label2 ... Two of three".

I have not tested this, but I was solely basing it on the way it reads out loud for normal folders in Windows.

**Comment by beccamc** [comment]
![image](https://user-images.githubusercontent.com/9122518/106542113-d08bed00-64c8-11eb-83f7-f95bd3caff07.png)

**Comment by SaisreeDandamudi** [comment]
@beccamc 
Verified the bug in the latest build and the issue is partially fixed.
Narrator is reading Label1 1 of 3 expanded.
Now it is understandable for screen reader users but  I think that it should not be announced expanded/collapsed
Reading Expand/collapse may give that idea to the user that these are actionable content.
So reading expanded is correct in this case as it is just a non-actionable text.

Since this is Sev4 bug. As per [C & AI Bug Severity process ](https://microsoft.sharepoint.com/teams/cea11y/SitePages/Bug-Severity-Process.aspx)Sev4 bugs does not effect your SLA. So you can add them to your backlog

**Comment by beccamc** [comment]
Rohan is setting up a meeting with Dante on what the experience here should be.

**Comment by beccamc** [comment]
Current proposal: 

Sample Folder (Sample Folder, Level 1)
Sub Folder Label 1 (Sub Folder 1, Level 2)
Image 1 (Image file 1, Level 3)
Image 2 (Image file 2, Level 3)
Sub Folder Label 2 (Sub Folder 2, Level 2)
Sub Folder Label 3 (Sub Folder 3, Level 2)

**Comment by beccamc** [comment]
Here is the build for testing https://devdiv.visualstudio.com/_apis/resources/Containers/7859052/VSSetup.Dev16Insertion?itemPath=VSSetup.Dev16Insertion%2FMicrosoft.ML.ModelBuilder.vsix

**Comment by beccamc** [comment]
vzhuqin, you don't need to verify this one. @SaisreeDandamudi from the accessibility team will need to verify and close this. Thanks!

**Comment by beccamc** [comment]
Please use public build https://marketplace.visualstudio.com/items?itemName=MLNET.07

**Comment by beccamc** [comment]
@SaisreeDandamudi can you please verify this? Thanks!

**Comment by beccamc** [comment]
@NiharikaTipirishetty It seems like our old accessibility contact isn't here anymore. Could you validate this as well? You can check it on rel/candidate build 16.7.2.2141903.

**Comment by NiharikaTipirishetty** [comment]
Issue is fixed. In scan mode, Narrator is clearly announcing the levels of folders in example folder structure format. Hence closing this bug.


 ![1154_Narrator_fixed](https://user-images.githubusercontent.com/83570638/133405281-23fee855-44ee-4b96-90db-6483384add0a.png)
